mantidproject/mantid
# simpleapi and fitfunctions sphinx docs

I discovered that `mantid.simpleapi` and `mantid.fitfunctions` were not in the sphinx site. This adds them in. Since `simpleapi` is full of internals and is meant as a easy way to get the algorithms into python, that page is just a link to the normal list of algorithms.

**To test:**

Build the docs and look at `api/python/mantid` and see links to two new pages.

*There is no associated issue.*

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
